src/components/DropList.js

```diff
@@ -0,0 +1,26 @@
+import "./DropList.css"
+
+const DropList = (props) => {
+
+    const dropListItemClickHandler = (event) => {
```

Comment:
call it handelDropListItemClick -> function name should be a verb

Reply:
done .